dluhc-component-library/src/components/siteSelectionForm/components/DynamicForm.tsx

```diff
@@ -25,6 +27,9 @@ const getInputType = (formPageSchema: FormPageSchema) => {
   if (formPageSchema.type === "number") {
     return InputType.NumberInput;
   }
+  if (formPageSchema.type === "radio") {
```

Comment:
Worth noting "radio" isnt a valid jsonSchema type but until we add uiSchema this is a work around